mod-clans/src/main/java/gg/solarmc/loader/clans/ClanManager.java

```diff
@@ -186,5 +187,56 @@ public void deleteClan(Transaction transaction, Clan clan) {
         clans.invalidate(clan.getClanId());
     }
 
+    /**
+     * Gets x amount of the highest ranked clans by kills
+     * Ordered by highest to lowest.
+     * @param amount the amount of clans to fetch
+     * @return an ordered list (highest first, lowest last) of
+     * clans ranked by stat type.
+     */
+    public List<Integer> getTopClanKills(Transaction transaction, int amount) {
```

Comment:
Instead of returning raw integers, can this method and the other ones return an object containing the clan name?

```java
```suggestion
    public record ClanInfo(int clanId, String clanName) {}

    public List<ClanInfo> getTopClanKills(Transaction transaction, int amount) {
```

The same can be said for the other methods (deaths and assists).

Reply:
WIll do